### DISCUSSION TITLE
[NODRIVER] Possible to run in Spyder IDE?

### DISCUSSION THREAD

**Discussion by tomdav999** [ultrafunkamsterdam/undetected-chromedriver#2091] [Q&A]
I can't seem to run nodriver in the Spyder IDE.   I get "RuntimeError: asyncio.run() cannot be called from a running event loop".  I tried using `nest_asyncio.apply()` but this results in "NotImplementedError".  Has anyone figured out how to get nodriver to run in Sypder?

**Comment by Carragos** [comment]
Same issue, would love some insight on this!
